src/Sensors.hpp

```diff
@@ -81,6 +81,10 @@ class Sensors {
     // temperature offset (for final temp output)
     float toffset = 0.0;  
 
+    // Altitud compensation variable
+    float alt_comp = 0.0;
+    float altitude = 0.0;
```

Comment:
I don't understand why we need `altitude` variable? it is a local variable maybe?

Reply:
erased